docs/tractus-x-release.md

```diff
@@ -0,0 +1,131 @@
+## Intro
+
+Tractus-X operates on a quarterly release cycle. Each quarter, a new Tractus-X release following CalVer is released. Each Tractus-X release contains multiple products product_release.md.
```

Comment:
Tractus-X operates on a quarterly release cycle. Each quarter, a new tractus-x release following [CalVer] (URL) is released. Each tractus-x release contains multiple [products] (product_release.md).

Reply:
As mentioned in the other comment, i would write Tractus-X upper case.

Also pls be more specific what issues you see. Its weird to read and diff the sentence to find this out

Reply:
Btw. we are repeating ourselves here. Its the same sentence as in README.md

Reply:
no duplicates found. pls keep sentence.
pls update to:
Tractus-X operates on a quarterly release cycle. Each quarter, a new Tractus-X release following [CalVer] (URL) is released. Each Tractus-X release contains multiple [products] (product_release.md).
hint: URL links shown in normal brackets